src/App.js

```diff
@@ -1,24 +1,117 @@
-import logo from './logo.svg';
+import React, { useEffect, useState } from "react";
+import { Switch, Route, useHistory } from "react-router-dom";
 import './App.css';
+import 'antd/dist/antd.css';
+import HomePage from './pages/homePage';
+import PeoplePage from './pages/peoplePage';
+import Profile from './pages/profilePage';
+import { Layout, Menu, Button } from "antd";
+import {
+  MenuUnfoldOutlined,
+  MenuFoldOutlined,
+  UserOutlined,
+  VideoCameraOutlined,
+  UploadOutlined,
+} from "@ant-design/icons";
+const { Header, Sider, Content } = Layout;
 
-function App() {
+const routes = [
+  {
+    path: '/people',
+    component: PeoplePage,
+  },
+  {
+    path: '/profile',
+    component: Profile
+  },
+  {
+    path: '/',
+    component: HomePage,
+  },
+];
+
+function App({ component: Component, ...rest }, props) {
+  const [collapsed, setCollapsed] = useState(false);
+  const history = useHistory();
+
+  useEffect(() => {
+    history.push("/");
```

Comment:
Is it intentional? Otherwise, it is a bug. If you visit `/people`, it will redirect to `/`